.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-- 8.10
+- "8.10"
```

Comment:
Why this change?

Reply:
because it suddenly uses 8.1 if there are no quotes